tests/integration_test.rs

```diff
@@ -625,7 +642,8 @@ async fn test_detect_certificate_rotation() {
     let key_file = certs_dir.path().join("policy-server-key.pem");
 
     let hostname1 = "cert1.example.com";
-    let tls_data1 = create_cert(&hostname1);
+    let var_name = &hostname1;
+    let tls_data1 = create_cert(var_name);
```

Comment:
I'm a bit puzzled about this change, nothing else was touched inside of the test. Do we need it?

Reply:
the linter was complaining, i just applied the suggestion

Reply:
you are right, it seems i've applied the wrong code action, let me revert it